apps/rest-patch-stops-working-after-restart/rest-patch-stops-working-after-restart.py

```diff
@@ -0,0 +1,87 @@
+"""
+https://semi-technology.atlassian.net/browse/WEAVIATE-151
+"""
+
+import weaviate
+from loguru import logger
+import sys, traceback
+
+def create_weaviate_schema():
+    schema = {
+        "classes": [
+            {
+                "class": "PatchStopsWorkingAfterRestart",
+                "vectorizer": "none",
+                "vectorIndexType": "hnsw",
+                "invertedIndexConfig": {
+                  "bm25": {
+                    "b": 0.75,
+                    "k1": 1.2
+                  },
+                  "cleanupIntervalSeconds": 60,
+                  "stopwords": {
+                    "preset": "en"
+                  }
+                },
+                "properties": [
+                    {
+                        "dataType": [
+                            "string"
+                        ],
+                        "name": "description",
+                        "tokenization": "word",
+                        "indexInverted": True
+                    },
+                ]
+            },
+        ]
+    }
+    # add schema
+    if not client.schema.contains(schema):
+        client.schema.create(schema)
+
+def get_post(index, id: str, req_type):
```

Comment:
It's not clear to me why the function is returning a list of different types.
Since the class name and object_id are global variables It makes sense to use a function like
`def get_body(index, req_type):
   return  {
             "description": f"this is an update number: {index} with {req_type}"
         } `